webroot/css/toolbar.css

```diff
@@ -130,6 +131,13 @@ p {
 	color: #ab47bc;
 }
 
+#cake-version {
```

Comment:
There are some of those already bot.
Mark let me know if you want me to transform this into a class. no problem